js/src/components/paid-ads/ads-campaign/ads-campaign.js

```diff
@@ -89,6 +90,8 @@ export default function AdsCampaign( {
 				{ showCampaignPreviewCard && <CampaignPreviewCard /> }
 			</BudgetSection>
 
+			<EuRegulationsSection context={ context } />
```

Comment:
Nitpick, but this would be a good use case for wrapping this AdsCampaign return in a Context provider so we can make use of the useContext hook rather than prop drilling here.